ecllibrary/std/File.ecl

```diff
@@ -1040,4 +1040,29 @@ EXPORT varstring GetDefaultDropZone() :=
 EXPORT dataset(FsDropZoneRecord) GetDropZones() :=
     lib_fileservices.FileServices.GetDropZones();
 
+/**
+ * Return the expire days property of the specified logical filename.
+ *
+ * @param lfn           The name of the logical file.
+ 
+ 
+ * @return  	        Value of @expiredays properties of the file or -1 if that property not exists (never expire).
+ *
+ */
+
+EXPORT integer4 GetExpireDays(varstring lfn) := 
+	lib_fileservices.Fileservices.GetExpireDays(lfn);
+	
+
+/**
+ * Set the expire days property of the specified logical filename.
+ *
+ * @param lfn           The name of the logical file.
+ * @param expireDays    Number of days before file expired. Use -1 to remove 
```

Comment:
see other comment re. ClearExpiryDays, and worth comment mentioning what 0 means.

Reply:
ClearExpireDays() function added, comment added to SetExpireDays() funtion parameter.